### DISCUSSION TITLE
Read Text message

### DISCUSSION THREAD

**Discussion by bilo2000** [gabrieldwight/Whatsapp-Business-Cloud-Api-Net#65] [General]
Hi,

i am trying to read the recieved Text from text message using this code

  
                        textSenderSelection = textMessage.SingleOrDefault().Text.ToString();

for some reason it got empty or null, what is the issue

**Comment by gabrieldwight** [comment]
You should check the json properties received in the webhook and try to deserialize the properties since whatsapp always dynamic webhook responses

**Reply by bilo2000** [reply]
Thanks
